eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -457,16 +458,17 @@ jobs:
           packageStepDescription: Runtime Deps, Runtime, Framework Packs installers
```

Comment:
I think this can be included in the skip, since runtime packs don't have installers. (The desc is referring to the shared framework runtime, and the targeting and apphost framework packs.)

Reply:
I need to trust the comments more when it talks about installers ;-)

Reply:
I can easily imagine parsing it as saying `Runtime [...] Packs installers`, so at the same time trusting them less is also reasonable. 😄